gradoop-examples/src/main/java/org/gradoop/examples/subgraph/SubgraphExample.java

```diff
@@ -52,7 +54,8 @@ public static void main(String[] args) throws Exception {
     // Create Gradoop config.
     GradoopFlinkConfig config = GradoopFlinkConfig.createConfig(executionEnvironment);
     // Read the input graph.
-    LogicalGraph inputGraph = new CSVDataSource(DATA_PATH, config).getLogicalGraph();
+    LogicalGraph inputGraph = new CSVDataSource(
```

Comment:
The logic of decoding the input path here can be done during the initialization of the `DATA_PATH` parameter.